src/runtime_src/tools/xclbinutil/RapidJsonUtilities.h

```diff
@@ -28,12 +28,15 @@
 
 namespace XclBinUtilities {
 
-typedef enum {
-  DType_UNKNOWN,
-  DType_INTEGER,
-  DType_TEXT_STRING,
-  DType_BYTE_STRING
-} DType;
+enum class DType {
+  UNKNOWN,
+  INTEGER,
+  TEXT_STRING,
+  BYTE_STRING,
+  HEX_BYTE_STRING,
+  BYTE_FILE,
+  ENUMERATION,
```

Comment:
enum classes should use lower case enumerators.   The upper case ones still run then preprocessor risk of conflicting with brain-dead code that #defines say UNKNOWN to something odd.

Reply:
done